spec/Section 3 -- Type System.md

```diff
@@ -232,6 +232,153 @@ Schema extensions have the potential to be invalid if incorrectly defined.
 2. Any non-repeatable directives provided must not already apply to the
    original Schema.
 
+### Schema Coordinates
+
+Schema Coordinates are human readable strings that uniquely identify an element defined in a GraphQL Schema.
+
+**Definition**
+
+SchemaCoordinate:
```

Comment:
I think we might want a `Note:` below this definition making it clear that `SchemaCoordinate` is not part of a `GraphQLDocument` and mentions the expected use case